docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -324,6 +324,35 @@ and needs to be manually advanced):
   links to the security registrar (http://www.silverstripe.org/download/security-releases)
   match the pages saved in draft.
 
+#### Basing a new release on a previous one (tweak releases)
+
+Commonly a stable release will need to mirror the contents of the release
+candidate that preceded it, sometimes with a small set of additional commits.
+However, running the standard `cow release` command will create a release that
+includes all the latest commits on the branches it targets, which can include
+unaudited code. A **tweak release** includes only the commits present in the
+previous tagged release by default, and can optionally include additional
+commits when necessary. To create one, use the `release:checkout-tagged-base`
+command:
+
+1. `cow release:create <new-version>` to create the new release.
+2. `cow release:plan <new-version>` to generate a plan for the new release.
+3. `cow release:checkout-tagged-base <new-version>` to shift all of the modules
+  to the commit in the release branch that matches the previous version tag.
+  * **How?** This command finds the last common commit between the previous
+    version tag and the release branch, and shifts the HEAD to that commit.
+4. `cherry-pick` any extra commits that need to be included in the release onto
```

Comment:
what branch would we cherry-pick from? 4.x-dev ? or an unmerged branch?

Reply:
In most cases, you'd be cherry-picking a specific commit from the related release branch (e.g. a fix applied to 4.5 to go into a 4.5.1 release) - the reason we `detach-tagged-base` and `cherry-pick` is to avoid including any commits between the previous tag and the exact commit you're after. This could be the 4.5 branch on the public repo for a generic fix, or the 4.5 branch on the related silverstripe-security fork for a security fix.

If you attach a fork to the module in the release folder, you can cherry-pick from it, yep.